src/scaffoldmaker/meshtypes/meshtype_3d_lung1.py

```diff
@@ -184,42 +232,131 @@ def generateBaseMesh(cls, region, options):
                             nodeIdentifiers.pop(4)
                             nodeIdentifiers.pop(0)
                             eft = eftWedgeCollapseXi1_15
-                        elif (e2 == 0) and (e1 == elementsCount1-1):
+                        elif (e2 == 0) and (e1 == lelementsCount1-1):
```

Comment:
My preferred formatting is to have spaces around most operators; multiply and divide are sometimes the exception. Also use brackets.
Hence:
``elif (e2 == 0) and (e1 == (lElementsCount1 - 1)):``